src/utils.js

```diff
@@ -132,23 +131,9 @@ export function shuffleArray(original) {
   return array
 }
 
-// Makes “gu” a CSS unit in a string, e.g.
-// cssgu`10px 2gu 4gu`
-export function cssgu(strings, ...substitutions) {
-  return strings
-    .map((str, i) => str + (substitutions[i] || ''))
-    .join('')
-    .replace(/([0-9]+(?:\.[0-9]+)?)gu/g, (match, value) => {
-      value = parseFloat(value)
-      return isNaN(value) ? match : `${value * GU}px`
-    })
-}
```

Comment:
:wave: :dancers:

Reply:
is that something that you used before having the grid unit?

Reply:
Yes, this was _just_ before the grid unit was introduced (we released the activity panel late April, and began introducing the GU in mid May).